studio/localStores/storageExplorer/StorageExplorerStore.js

```diff
@@ -1176,7 +1176,7 @@ class StorageExplorerStore {
     const files = await this.getAllItemsAlongFolder(folder)
     await this.deleteFiles(files, isDeleteFolder)
 
-    const isFolderOpen = this.openedFolders[this.openedFolders.length - 1].name === folder.name
+    const isFolderOpen = this.openedFolders[this.openedFolders.length - 1]?.name === folder.name
```

Comment:
I don't see why this line is changed?

Reply:
This throws an error when you attempt to delete a closed folder (using the menu items on the list).
i.e With no folders opened, `this.openedFolders` would be an empty array, and `this.openedFolders[this.openedFolders.length - 1]` would be `undefined`, which throws and error when you try to access `.name` property of `undefined`.